src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubSCMNavigator.java

```diff
@@ -95,14 +102,13 @@ public String getPattern() {
             listener.getLogger().format("Must specify user or organization%n");
             return;
         }
-        String apiUrl = null; // TODO GHE
-        StandardCredentials credentials = Connector.lookupScanCredentials(observer.getContext(), apiUrl, scanCredentialsId);
+        StandardCredentials credentials = Connector.lookupScanCredentials(observer.getContext(), null /* TODO apiUri */, scanCredentialsId);
```

Comment:
Do we want to include this now? What happen if I'm using a domain for my GHE instance **but** I want to use a specific credentials? As far as a I know, this will filter for this domain.